README.md

```diff
@@ -411,10 +411,22 @@ using a combination of `bwa` and `samtools` to output a BAM file instead of a SA
     - `*.fastq.gz` and NOT `*.fastq`
     - `*.bam` and NOT `*.sam`
 
-- Where applicable, each module command MUST emit a file `<SOFTWARE>.version.txt` containing a single line with the software's version in the format `<VERSION_NUMBER>` or `0.7.17` e.g.
+- Where applicable, each module command MUST emit a file `versions.yml` emitting the version number for each tool executed by the module, e.g.
 
     ```bash
-    echo \$(bwa 2>&1) | sed 's/^.*Version: //; s/Contact:.*\$//' > ${software}.version.txt
+    cat <<-END_VERSIONS > versions.yml
```

Comment:
Maybe we need some docs about why we are using HERE DOCUMENTS here?

Reply:
Do you mean something like this? 

> We chose a [HEREDOC](https://tldp.org/LDP/abs/html/here-docs.html) over piping into the versions file line-by-line as we believe the latter makes it easy to accidentally overwrite the file. Moreover, the exit status of the sub-shells evaluated in within the HEREDOC is ignored, ensuring that a tool's version command does not erroneously terminate the module.

Reply:
This will need to be moved to the website now since we have moved the docs but looks great 👍🏽

Reply:
resolved in https://github.com/nf-core/nf-co.re/pull/877